source/modules/configuration/general/general_configuration.rst

```diff
@@ -0,0 +1,32 @@
+General configuration
+=====================
+
+This tab permits to customize main application appearance.
+
+.. image:: images/general_configuration.png
+
+Description of fields
+---------------------
+
+* **URL of the application**: used in different links provided in the application, in notifications, and for the API.
+  Its defined at install time depending on the URL that has been used for it.
+* **Text in the login box**: specific text to be displayed at the top of the login block
+* **Allow FAQ anonymous access**: allows :abbr:`Frequently Asked Question <FAQ>` access without any login in GLPI. A link to access the FAQ will be displayed below the login box when enabled.
+* **Default search results limit**: maximum number of search results displayed at once on all search pages
+* **Default characters limit**: maximum number of displayed characters visible in search results. If you display a ticket content for example, ill will be truncated on this number of characters
```

Comment:
```suggestion
* **Default characters limit**: maximum number of displayed characters visible in search results. If you display a ticket content for example, it will be truncated on this number of characters
```